integrations/vllm_plugin/vllm_tt/model_runner.py

```diff
@@ -1842,12 +1867,10 @@ def reset_dynamo_cache(self):
             )
             compiled_model.compiled_codes.clear()
 
-    @torch.compile(backend="tt", fullgraph=True, dynamic=False)
+    torch.compile(backend="tt", fullgraph=True, dynamic=False)
+
     def select_hidden_states(self, hidden_states, indices_do_sample):
```

Comment:
Either use (if you want to compile)
```suggestion
    @torch.compile(backend="tt", fullgraph=True, dynamic=False)
    def select_hidden_states(self, hidden_states, indices_do_sample):
```
Or (if you don't want to compile). 
```suggestion
    # @torch.compile(backend="tt", fullgraph=True, dynamic=False)
    def select_hidden_states(self, hidden_states, indices_do_sample):
```

Reply:
Yeah I missed `@`. Thanks!